src/services/ContributingDeveloperAuditService/ContributingDeveloperAuditService.ts

```diff
@@ -0,0 +1,25 @@
+import { ScmType } from "../../enums";
```

Comment:
This service is under its own folder since its quite complex compared to the others, but probably we should also put the others into their own folders just to keep the structure the same even tho they are only one file.

Reply:
Makes sense - I'd go ahead and do that now.

Reply:
It's ok to wait too - the pattern is to use one file until you need to break out sub-requirements (like this one), then that file becomes the folder name